railties/lib/rails/generators/app_base.rb

```diff
@@ -760,11 +760,11 @@ def keep_file(destination)
       end
 
       def user_default_branch
-        @user_default_branch ||= `git config init.defaultbranch`
+        @user_default_branch ||= `git config init.defaultbranch`.strip
```

Comment:
I'm not sure how frequent this is, but if a user doesn't have git installed and runs `rails new my_app --skip-git` this will now crash the process.

Reply:
That's interesting, maybe `--skip-git` should imply `--skip-ci`

Reply:
Maybe an even easier solution: `$default-branch`

https://github.blog/changelog/2020-07-22-github-actions-better-support-for-alternative-default-branch-names/